.latexindent.yaml

```diff
@@ -0,0 +1,4 @@
+onlyOneBackUp: 1
+modifyLineBreaks:
+    textWrapOptions:
+        columns: 100
```

Comment:
This file contains the `latexindent` config (specifically what we want different from the default).
1. I left the default of an indent being a single tab (`\t`) instead of using spaces. This cause be changed with the `defaultIndent` property.
2. Max line length is 100 characters. With the way it does formatting, it is possible for lines to be longer than 100 characters as it does multiple passes and does different updates each pass.

Let me know if you want it to be different. I know there are accessibility advantages to using tabs instead of spaces. Also [prettier plans to switch to tabs in v3.0](https://github.com/prettier/prettier/issues/7475#issuecomment-782939718).